platform-sdk/swirlds-common/src/main/java/com/swirlds/common/config/BasicConfig.java

```diff
@@ -251,4 +263,10 @@ public record BasicConfig(
         @ConfigProperty(value = "socketIpTos", defaultValue = "-1") int socketIpTos,
         @ConfigProperty(value = "maxIncomingSyncsInc", defaultValue = "1") int maxIncomingSyncsInc,
         @ConfigProperty(value = "maxOutgoingSyncs", defaultValue = "2") int maxOutgoingSyncs,
-        @ConfigProperty(value = "logPath", defaultValue = "log4j2.xml") Path logPath) {}
+        @ConfigProperty(value = "logPath", defaultValue = "log4j2.xml") Path logPath,
+        @ConfigProperty(value = "syncAsProtocolEnabled", defaultValue = "true") boolean syncAsProtocolEnabled,
```

Comment:
This should be disabled by default before it gets merged. We will only flip it on by default once we know we want to enable it in production environments.

Reply:
This means that when this merges, we need to update the nightly JRS tests to turn it on for nightly test coverage.